oci/client/list.go

```diff
@@ -66,8 +69,8 @@ func (c *Client) List(ctx context.Context, url string, opts ListOptions) ([]Meta
 	}
 
 	for _, tag := range tags {
-		// exclude cosign signatures
-		if strings.HasSuffix(tag, ".sig") {
+		// ignore cosign artifacts by default
+		if isCosignArtifact(tag) && !opts.IncludeCosignArtifacts {
```

Comment:
I'm temped to make it a method on `ListOptions`, but maybe it'd be pointless - wdyt?
```
func (o ListOptions) ignore(tag string) bool {
      for _, suffix := range []string{".att", ".sbom", ".sig"} {
 		if strings.HasSuffix(tag, suffix) {
 			return !o.IncludeCosignArtifacts
 	        }
      }
      return false
 }
```
So it's be called like `if opts.ignore(tag)`...

Reply:
We'll need the `IsCosignArtifact` func in other places of Flux where `List` is not used. So I would keep it as stand alone public function.